src/main/java/frc/trigon/robot/subsystems/climber/placeholderclimber/PlaceholderClimberConstants.java

```diff
@@ -0,0 +1,95 @@
+package frc.trigon.robot.subsystems.climber.placeholderclimber;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.mechanisms.DifferentialMechanism;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import frc.trigon.robot.constants.RobotConstants;
+
+public class PlaceholderClimberConstants {
+    static final boolean ENABLE_FOC = true;
+    private static final int
+            RIGHT_MOTOR_ID = 0,
+            LEFT_MOTOR_ID = 0;
+    private static final InvertedValue
+            RIGHT_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive,
+            LEFT_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Coast;
+    private static final boolean ARE_MOTORS_ALIGNED = true;
+    private static final double GEAR_RATIO = 1;
+    private static final double
+            MOTION_MAGIC_ACCELERATION = 200,
+            MOTION_MAGIC_VELOCITY = 160,
+            MOTION_MAGIC_JERK = 1600;
+    private static final double
+            AVERAGE_P = 0,
+            AVERAGE_I = 0,
+            AVERAGE_D = 0,
+            AVERAGE_KA = 0,
+            AVERAGE_KG = 0,
+            AVERAGE_KS = 0,
+            AVERAGE_KV = 0,
+            DIFFERENTIAL_P = 0,
+            DIFFERENTIAL_I = 0,
+            DIFFERENTIAL_D = 0;
+
+    static final TalonFX
+            RIGHT_MOTOR = new TalonFX(RIGHT_MOTOR_ID, RobotConstants.CANIVORE_NAME),
+            LEFT_MOTOR = new TalonFX(LEFT_MOTOR_ID, RobotConstants.CANIVORE_NAME);
```

Comment:
this should be private

Reply:
but I need the motors for the stop method. there isn't a stop method in the differential mechanism

Reply:
Right, my bad.